src/pages/test-page/TestPage.jsx

```diff
@@ -286,6 +386,62 @@ function Test() {
 					/>
 				</Stack>
 				<ActionAreaCard />
```

Comment:
Aici in `ActionAreaCard` au fost implementate card-urile, avem afisate mai multe dintre ele pe pagina de test. Asigura-te ca te uiti peste aceste componenete. Refoloseste componenta din `components/shared/card` drept baza